Big-Spender/readme.md

```diff
@@ -68,51 +68,52 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+SELECT amount FROM spends WHERE (LOWER(description) LIKE '%fees%' OR LOWER(description) LIKE '%fee%');
```

Comment:
Could you have just done `WHERE LOWER(description) LIKE '%fee%'` to replicate this OR statement?

What about items with descriptions like "Very Important Coffee Machine" or "2 Replacement Feet for Chair"?

Reply:
I should have used ILIKE instead of LIKE to make query case-insensitive.